fineract-provider/src/main/java/org/apache/fineract/infrastructure/creditbureau/data/CreditBureauLoanProductMappingData.java

```diff
@@ -104,7 +104,7 @@ public long getStalePeriod() {
         return this.stalePeriod;
     }
 
-    public boolean isisActive() {
+    public boolean isIs_active() {
```

Comment:
isActive() should be sufficient

Reply:
Resolving, as this looks like it has been addressed. Please Resolve open discussion items yourself in the future, as it makes it easier and faster for me to review pending pull requests.